content/docs/tutorials/getting-started-with-cert-manager-on-azure-kubernetes-service-using-lets-encrypt-for-ssl/README.md

```diff
@@ -0,0 +1,806 @@
+
+
+As you can see there are some variables in the `clusterissuer-lets-encrypt-staging.yaml` which need to be filled in before we apply it;
+most have been defined earlier in this tutorial but you need to set the following:
+
+```bash
+export EMAIL_ADDRESS=<email-address> # ❗ Replace this with your email address
+export AZURE_SUBSCRIPTION=<your-subscription-or-billing-account>  # ❗ Replace this with your Azure account name
+```
+
+Now use `envsubst` to fill in the variables and pipe it into `kubectl apply`, as follows:
+
+```bash
+export AZURE_SUBSCRIPTION_ID=$(az account show --name $AZURE_SUBSCRIPTION --query 'id' -o tsv)
+envsubst < clusterissuer-lets-encrypt-staging.yaml | kubectl apply -f  -
+```
+
+You can check the status of the ClusterIssuer:
+
+```bash
+kubectl describe clusterissuer letsencrypt-staging
+```
+
+Example output
+
+```console
+Status:
+  Acme:
+    Last Registered Email:  firstname.lastname@example.com
+    Uri:                    https://acme-staging-v02.api.letsencrypt.org/acme/acct/77882854
+  Conditions:
+    Last Transition Time:  2022-11-29T13:05:33Z
+    Message:               The ACME account was registered with the ACME server
+    Observed Generation:   1
+    Reason:                ACMEAccountRegistered
+    Status:                True
+    Type:                  Ready
+```
+
+> ℹ️ Let's Encrypt uses the Automatic Certificate Management Environment (ACME) protocol
+> which is why the configuration above is under a key called `acme`.
+>
+> ℹ️ The email address is only used by Let's Encrypt to remind you to renew the certificate after 30 days before expiry. You will only receive this email if something goes wrong when renewing the certificate with cert-manager.
+>
+> ℹ️ The Let's Encrypt production issuer has [very strict rate limits](https://letsencrypt.org/docs/rate-limits/).
+> When you're experimenting and learning, it can be very easy to hit those limits. Because of that risk,
+> we'll start with the Let's Encrypt staging issuer, and once we're happy that it's working
+> we'll switch to the production issuer.
+>
+> 📖 Read more about [configuring the ACME Issuer](../../configuration/acme).
+>
+
+
+## Re-issue the Certificate using Let's Encrypt
+
+Patch the Certificate to use the staging ClusterIssuer:
+
+```bash
+kubectl patch certificate www --type merge  -p '{"spec":{"issuerRef":{"name":"letsencrypt-staging"}}}'
+```
+
+That should trigger cert-manager to renew the SSL certificate:
+Use `cmctl` to check:
+
+```bash
+cmctl status certificate www
+cmctl inspect secret www-tls
+```
+
+And finally, when the new SSL certificate has been issued, you can restart the web server to use the new SSL certificates:
+
+```bash
+kubectl rollout restart deployment helloweb
+```
+
+You should once again be able to connect to the website, but this time you will see the Let's Encrypt staging certificate:
+
+```
+$ curl -v --insecure https://www.$DOMAIN_NAME
```

Comment:
suggestion: This is missing a type - it should be ```text I think.

suggestion: Is it worth pointing out (again) that we're using `--insecure` because it's LE staging, and that we'll get to a secure cert really soon?